Robot2019/src/main/java/frc/robot/commands/Drive.java

```diff
@@ -0,0 +1,61 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.Robot;
+import frc.robot.RobotMap;
+import frc.robot.OI;
+
+/**
+ * An example command.  You can replace me with your own command.
+ */
+public class Drive extends Command {
+
+  public Drive() {
+    // Use requires() here to declare subsystem dependencies
+    requires(Robot.motorSubsystem);
+  }
+
+  // Called just before this Command runs the first time
+  @Override
+  protected void initialize() {
+  }
+
+  // Called repeatedly when this Command is scheduled to run
+  @Override
+  protected void execute() {
+
+    double yVal=OI.stick.getRawAxis(1);
+    double xVal=OI.stick.getRawAxis(4);
+
+    Robot.motorSubsystem.drive (yVal , xVal);
```

Comment:
Just a small style note, typically there isn't a space between the method name and the parameter list.